models/orderDao.js

```diff
@@ -0,0 +1,168 @@
+const {appDataSource} = require('./datasource')
+const middleErr = require('../middleware/error')
+
+// 장바구니에 담긴 상품 보여주기
+const busket = async (verifiedEmail) => {
+    try {
+        const userId = await appDataSource.query(`
+        select id from users where email = '${verifiedEmail}'
+        `)
+        const extractedId = userId[0].id
+        
+        const result = await appDataSource.query(`
+        SELECT p.id AS product_id, p.price AS product_price, p.name AS product_name, b.logo AS product_logo, bsk.quantity AS basket_quantity, b.name AS brand_name
+        FROM users u
+        JOIN basket bsk ON u.id = bsk.user_id
+        JOIN products p ON bsk.product_id = p.id
+        JOIN brands b ON p.brand_id = b.id
+        JOIN pay_status ps ON bsk.status_id = ps.id
+        WHERE u.id = '${extractedId}' AND ps.id = 1;       
+        `)
+        return result
+        
+    } catch (err) {
+        console.log(err)
+        middleErr.error(500 , 'INVALID_DATA_INPUT' )
+    }
+}
+
+//장바구니 상품 수량추가
+const addProduct = async (verifiedEmail, productId) => {
```

Comment:
```suggestion
const increaseBasketQuantity = async (verifiedEmail, productId) => {
```
- `addProduct`라는 함수 이름은 해당 기능이 "상품을 등록하는 과정"이라고 오해받을 수 있을 확률이 있습니다. 장바구니에 상품을 추가하여 수량을 증가시키는 것이라면 보다 명확한 함수 이름을 작성하셔야 합니다.
- 아래 수량 감소 과정도 마찬가지이니 수정해 주세요